src/DynamoPublish/DynamoPublishExtension.cs

```diff
@@ -21,22 +31,27 @@ public string Name
 
         public void Startup(ViewStartupParams p)
         {
-            // Some start up stuff.
+            publishModel = new PublishModel(p.AuthProvider, p.CustomNodeManager);
+            publishViewModel = new PublishViewModel(publishModel);
         }
 
         public void Loaded(ViewLoadedParams p)
         {
-            // Some loaded stuff.            
```

Comment:
Keep in mind that, from the way you've structured things, it is possible that `publishViewModel` will be null.  That is because there are exceptions in the constructor for `PublishModel`.  That's okay, but you'll need to do a null check here as a result...